pkg/api/customization/kontainerdriver/formatter.go

```diff
@@ -26,15 +26,27 @@ func NewFormatter(manangement *config.ScaledContext) types.Formatter {
 	return format.Formatter
 }
 
-const clusterByKontainerDriverKey = "clusterbyKontainerDriver"
+const clusterByGenericEngineConfigKey = "genericEngineConfig"
 
-func clusterByKontainerDriver(obj interface{}) ([]string, error) {
+// IndexFunction that uses cluster's genericEngineConfig map[string]interface{}
+// to lookup the driverName
+func clusterByGenericEngineConfig(obj interface{}) ([]string, error) {
```

Comment:
The name should stay the same, this isn't doing an index by the config, it is still an index on the driver